charts/zot/templates/deployment.yaml

```diff
@@ -144,3 +158,8 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.dnsConfig }}
+      dnsConfig:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      dnsPolicy: {{ .Values.dnsPolicy }}
```

Comment:
I'm curious about the reasoning of not having dnsPolicy shielded with a `with` block like for the others. If in the future Kubernetes default ever changes it might be unexpected to have a default specified in the values of the helm chart?

Reply:
I was just following convention from other public helm charts that support this feature. For example the argocd helm chart has the dnsConfig in a with block but the dnsPolicy is not https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-application-controller/deployment.yaml